src/baseClient.ts

```diff
@@ -84,6 +84,35 @@ export class ClasschartsClient {
       }
     );
   }
+  
+  /**
+   * Returns all of the activities from and to the indicated dates.
+   * An abstraction on top of the getActivity method, it fetches everything in the given interval instead of returning a fragment.
+   */
+  async getFullActivity(from: string, to: string, client: ClasschartsClient) {
```

Comment:
```suggestion
  async getFullActivity(options:getFullActivityOptions) {
```
I'd prefer it if we could use a object like the other functions